src/evo/providertx.h

```diff
@@ -195,9 +195,7 @@ class CProUpRevTx
     // these are just informational and do not have any effect on the revocation
     enum {
         REASON_NOT_SPECIFIED = 0,
-        REASON_TERMINATION_OF_SERVICE = 1,
-        REASON_COMPROMISED_KEYS = 2,
-        REASON_CHANGE_OF_KEYS = 3,
+        REASON_CHANGE_OF_KEYS = 1,
```

Comment:
Can't redefine them that easy, see https://github.com/dashpay/dips/blob/master/dip-0003.md#appendix-a-reasons-for-self-revocation-of-operators. I would keep all values here for consistency, even though they are unused (i.e. revert 8ea0bbbc7b8ab1e779c02ae1dd029c11a9817d2e).

Reply:
As this is the last commit in this PR, I'd prefer to remove the whole commit instead of reverting it afterwards.